shell/components/ClusterIconMenu.vue

```diff
@@ -76,6 +76,7 @@ export default {
         style="enable-background:new 0 0 100 100;"
         xml:space="preserve"
       >
+        <title>{{ t('nav.ariaLabel.clusterIcon') }}</title>
```

Comment:
Feels like this one should contain something about it being the icon for the local cluster

Reply:
True! I thought that we could have more icons on clusters, but I've just double-checked we can't. updated the string and key